apm-server/README.md

```diff
@@ -40,17 +41,26 @@ See [supported configurations][] for more details.
 
 ## Installing
 
-This chart is tested with the latest 7.9.2 version.
+This chart is tested with the latest 7.9.3-SNAPSHOT version.
 
 ### Install released version using Helm repository
 
 * Add the Elastic Helm charts repo:
 `helm repo add elastic https://helm.elastic.co`
 
 * Install it:
-  - with Helm 2: `helm install --name apm-server --version 7.9.2 elastic/apm-server`
-  - with [Helm 3 (beta)][]: `helm install apm-server --version 7.9.2 elastic/apm-server`
+  - with Helm 2: `helm install --name apm-server --version <version> elastic/apm-server`
+  - with [Helm 3 (beta)][]: `helm install apm-server --version <version> elastic/apm-server`
+
+### Install development version from a branch
+
+* Clone the git repo: `git clone git@github.com:elastic/helm-charts.git`
 
+* Checkout the branch : `git checkout 7.9`
+
+* Install it:
+  - with Helm 2: `helm install --name apm-server ./helm-charts/apm-server`
```

Comment:
it should be specified in this section that it is also using snapshot docker images to avoid users confusion.

Reply:
Note that this already specified in https://github.com/elastic/helm-charts/pull/831/files#diff-ae72c3cb6c31c49e02d8b7896a43c68fR44:

>This chart is tested with the latest 7.9.3-SNAPSHOT version.

However, that may not be clear enough for users

Reply:
we may add `--set imageTag=<version>` in the command...